src/coreclr/src/tools/crossgen2/ILCompiler.ReadyToRun/Compiler/DependencyAnalysis/ReadyToRunCodegenNodeFactory.cs

```diff
@@ -99,11 +99,11 @@ public ISymbolNode ReadyToRunHelperFromTypeLookup(ReadyToRunHelperId id, Object
             return _genericReadyToRunHelpersFromType.GetOrAdd(new ReadyToRunGenericHelperKey(id, target, dictionaryOwner));
         }
 
-        private NodeCache<ReadOnlyDataBlobKey, BlobNode> _readOnlyDataBlobs;
+        private NodeCache<ReadOnlyDataBlobKey, SettableBlobNode> _readOnlyDataBlobs;
 
-        public BlobNode ReadOnlyDataBlob(Utf8String name, byte[] blobData, int alignment)
+        public SettableBlobNode ReadOnlyDataBlob(Utf8String name, int alignment)
```

Comment:
Could you rename this method to `SettableReadOnlyDataBlob`? We might want to have the non-settable version back at some point.

Reply:
Thanks. Renamed.